src/ConfigWidgets/StopSymbolDownloadDialog.ui

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>StopSymbolDownloadDialog</class>
+ <widget class="QDialog" name="StopSymbolDownloadDialog">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>400</width>
+    <height>256</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Stop Symbol Download</string>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1,4,1,1,0">
+   <item>
+    <widget class="QLabel" name="label">
+     <property name="text">
+      <string>Symbols for this module are still downloading</string>
+     </property>
+    </widget>
+   </item>
+   <item>
+    <widget class="QLabel" name="moduleLabel">
+     <property name="text">
+      <string></string>
+     </property>
+    </widget>
+   </item>
+   <item>
+    <widget class="QLabel" name="label_2">
+     <property name="text">
+      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Once downloaded, these symbols will be cached locally and will load quickly for subsequent sessions.&lt;br/&gt;If you don't need to load symbols for this module, you can stop this download.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+     </property>
+     <property name="wordWrap">
+      <bool>true</bool>
+     </property>
+    </widget>
+   </item>
+   <item>
+    <widget class="QCheckBox" name="rememberCheckBox">
+     <property name="text">
+      <string>Never download symbols for this module</string>
```

Comment:
"Never **automatically** download symbols...?"

This is about automatic downloads, right?

Reply:
done